bundle/tests/environment_overrides_test.go

```diff
@@ -29,10 +29,7 @@ func TestEnvironmentOverridesResourcesStaging(t *testing.T) {
 	b := loadTarget(t, "./environment_overrides/resources", "staging")
 	assert.Equal(t, "staging job", b.Config.Resources.Jobs["job1"].Name)
 
-	// Overrides are only applied if they are not zero-valued.
-	// This means that in its current form, we cannot override a true value with a false value.
-	// Note: this is not desirable and will be addressed by representing our configuration
-	// in a different structure (e.g. with cty), instead of Go structs.
-	assert.Equal(t, true, b.Config.Resources.Pipelines["boolean1"].Photon)
```

Comment:
Note: target override setting a boolean to false now works.

Remaining: confirm setting string to empty, integer to 0, etc.